lib/MetaCPAN/Query/Role/Common.pm

```diff
@@ -0,0 +1,11 @@
+package MetaCPAN::Query::Role::Common;
+
+use Moose::Role;
+use MetaCPAN::Types qw( Str );
+
+has es => ( is => 'ro', );
+
+has index_name => ( is => 'ro', );
```

Comment:
Do these attributes still need an `isa` and maybe `required`?

Reply:
will add that later... just needed that merged for Joel's coming suggester changes that depend on this change.